fossology/fossology
# feat(swh): Allow API token

## Description

1. Allow setting authentication token for REST call.
2. Move configuration from `.conf` file to `sysconf` table.

### Changes

1. Add `token` key for `Bearer` token authentication.
2. Move `conf` elements to `src/lib/php/common-sysconfig.php`.

## How to test

1. Run the agent on an upload.
2. Check if it finishes without error.
3. Try running the agent on stage (see #1836)
4. Load an upload without SWH agent's results, there should be no warning in apache error log.

Closes #1836
